functionary/functionary/settings/unicorn_.py

```diff
@@ -0,0 +1,3 @@
+UNICORN = {
```

Comment:
This probably happens naturally once the unicorn stuff is moved into the ui app, but just a note that I'd create a `ui_.py` settings file and put this there.

Reply:
done